### PR TITLE
Add Snyk scanning & monitoring

### PR DESCRIPTION
**Pull request by ryan-wren**

### CODE REVIEW

**Comment by ryan-wren** [comment]
Snyk scan failed due to legitimate vulnerabilities discovered in this repo

**Review by ssyms — APPROVED** [review]

**Review by vallieres — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -4,15 +4,9 @@ go 1.13
 
 require (
 	github.com/gin-contrib/cors v1.4.0
-	github.com/gin-gonic/gin v1.8.1
-	github.com/go-playground/validator/v10 v10.11.1 // indirect
-	github.com/goccy/go-json v0.9.11 // indirect
+	github.com/gin-gonic/gin v1.9.1
```
Comment (ryan-wren) [inline go.mod:7]: I only updated `github.com/gin-gonic/gin` from `v1.8.1` to `v1.9.1`. The other updates to this file and `go.sum` happened after running `go mod download` and `go mod tidy` on this branch.

**Comment by ryan-wren** [comment]
After my latest changes, the Snyk scans are passing during the job in the CircleCI workflow.

**Comment by ryan-wren** [comment]
I spoke with the CPEng team and this PR will be closed as this is only a sample / example repository. From CPEng:

> sample-go-cfd is a semi-auto generated codebase that is used as a support resource for docs' readers.

> This PR might make it harder someone to fully understand the example, and to contribute because of the context

So, for that reason I'll close this PR and not add Snyk scanning to this project.

**Pull request closed without merging**